### PR TITLE
Jb45810

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

File: sdk-setup/src/oomadvice [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+{
+    local fn=$1
+    local args=("${@:2}")
+
+    bash -c "$(declare -f $fn); $fn \"\$@\"" bash "${args[@]}"
+}
+
+set_defaults()
+{
+    OPT_OOM_ADJ=10
```
Comment (vigejolla) [inline sdk-setup/src/oomadvice]: How did you come up with the value 10?

Comment (martyone) [reply sdk-setup/src/oomadvice]: Just some positive value.

**Comment by martyone** [comment]
Rebased, updated conditions for running under vbox only

**Comment by vigejolla** [comment]
Still LGTM

**Pull request merged**